packages/cli/src/Logger.ts

```diff
@@ -0,0 +1,79 @@
+import * as winston from 'winston';
+import config = require('../config');
+
+import {
+	ILogger,
+	LogTypes,
+} from 'n8n-workflow';
+
+export class Logger implements ILogger {
```

Comment:
To keep this as a singleton:
```suggestion
class Logger implements ILogger {
```